user/enterprise/operations-manual.md

```diff
@@ -145,6 +145,31 @@ This can have various causes, including an automatic nvm update or a caching err
 
 This error is most likely caused by a self-signed certificate. During the build, the worker container attempts to fetch different files from the platform machine. If the server got provisioned with a self-signed certificate, curl doesn't trust this certificate and therefore fails. While we're working on resolving this in a permanent and sufficient way, currently the only solution is to install a certificate issued by a trusted Certificate Authority (CA). This can be a free Let's Encrypt certificate or any other trusted CA of your choice. We have a section in our [Platform Administration Tips](/user/enterprise/platform-tips/#Use-a-Lets-Encrypt-SSL-Certificate) page that walks you through the installation process using Let's Encrypt as an example.
 
+## Users are stuck in syncing state
+
+One or more user accounts are stuck in the `is_syncing = true` state. When you query the database, the number of users which are currently syncing is not decreasing over the time. Example:
+
+```bash
+travis_production=> select count(*) from users where is_syncing=true;
+ count
+-------
+  1027
+(1 row)
+```
+
+### Strategy
+
+Log into the platform machine via ssh. Then execute `travis console` to get into Travis' Ruby console. Via that console, reset the `is_syncing` flag for the users by executing the following command:
```

Comment:
Try to avoid excessive use of "via". 

Here you could say

1. Log into ...
2. Run `travis console` to open the Travis CI console
3. Reset the `is_syncing` for user accounts that are stuck by runing
    ```
    SQL
    ````